src/screens/Exchange/Exchange.js

```diff
@@ -43,6 +43,8 @@ import SelectorInput from 'components/SelectorInput';
 import Button from 'components/Button';
 import Spinner from 'components/Spinner';
 import DeploymentView from 'components/DeploymentView';
+import WalkthroughItem from 'components/Walkthrough/WalkthroughItem';
+import { WalkthroughItemParent } from 'components/Walkthrough/WalkthroughItemParent';
```

Comment:
How about export it and import it as single default?

Reply:
Yeah, could be done :) Will adjust